notebooks/01-lightcurves/01-Lightcurves.ipynb

```diff
@@ -56,9 +56,11 @@
    "source": [
```

Comment:
<p>I think we can keep this example, and have the takeaway be "you actually can't see the transits here"</p><p> </p><p>If you look closely, you'll notice two things:</p><ol><li>The amplitude of the oscillations increases over time</li><li>The period is roughly 1 day</li></ol><p> </p><p>This is due to stray light from Earth entering the field of view -- the day-period oscillation is actually Earth itself turning! I think that's a cool lesson to share, and it saves us needing to do further analysis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>changed the previous text and added most of this wording and mentioned to stay tuned for the next lesson on data quality</p>